reactor-core/src/main/java/reactor/core/publisher/FluxJust.java

```diff
@@ -73,6 +73,7 @@ public void subscribe(final CoreSubscriber<? super T> actual) {
 	@Override
 	public Object scanUnsafe(Attr key) {
 		if (key == Attr.BUFFERED) return 1;
+		if (key == Attr.RUN_STYLE) return Attr.RunStyle.SYNC;
```

Comment:
the inner `WeakScalarSubscription` should also be covered. Also, neightbour class `FluxJoin` isn't covered at all.